gremlin-go/driver/connection_test.go

```diff
@@ -8,19 +27,21 @@ import (
 )
 
 const runIntegration = false
+const testHost string = "localhost"
+const testPort int = 8182
 
 func TestConnection(t *testing.T) {
 	t.Run("Test connect", func(t *testing.T) {
 		if runIntegration {
-			connection := connection{"localhost", 8181, Gorilla, newLogHandler(&defaultLogger{}, Info, language.English), nil, nil, nil}
+			connection := connection{testHost, testPort, Gorilla, newLogHandler(&defaultLogger{}, Info, language.English), nil, nil, nil}
 			err := connection.connect()
 			assert.Nil(t, err)
 		}
 	})
 
 	t.Run("Test write", func(t *testing.T) {
```

Comment:
The test name is "write", but we do read operation (count). Suggesting to improve test naming convention to better reflect what test is doing.